### PR TITLE
Separate libpacemaker ordering functions into own source file

### PR DESCRIPTION
**Pull request by kgaillot**
This finishes separating functions related to specific constraint types into their own source files. pcmk_sched_constraints.c now contains only generic functions applying to all constraint types.

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , here's yet another low priority review :)

File: lib/pacemaker/pcmk_sched_bundle.c [hunk 1]
```diff
@@ -283,10 +284,12 @@ pcmk__bundle_internal_constraints(pe_resource_t *rsc,
...
-            pcmk__order_resource_actions(replica->ip, RSC_START, replica->container, RSC_START,
-                          pe_order_runnable_left|pe_order_preserve, data_set);
-            pcmk__order_resource_actions(replica->container, RSC_STOP, replica->ip, RSC_STOP,
-                          pe_order_implies_first|pe_order_preserve, data_set);
+            pcmk__order_starts(replica->ip, replica->container,
+                               pe_order_runnable_left|pe_order_preserve,
+                               data_set);
+            pcmk__order_stops(replica->container, replica->ip,
+                              pe_order_implies_first|pe_order_preserve,
+                              data_set);
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_bundle.c:292]: These are the only two that switched from pcmk__order_resource_actions to pcmk__order_*.  Was that intentional?

**Comment by clumens** [comment]
Aside from the one comment, this all looks good to me.

File: lib/pacemaker/pcmk_sched_bundle.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_bundle.c:292]: Yep, the latter are convenience macros wrapping the former, and this was a place they could be used but weren't

**Pull request merged**